templates/index.html

```diff
@@ -18,6 +18,42 @@ <h4 id="takeover-subtitle">Raspberry Pi的Ubuntu桌面，最新的桌面功能
   </div>
 </section>
 
+<section class="p-strip u-no-padding--bottom">
+  <div class="row">
+    <div class="col-6 col-medium-4">
+      <h2>现代化的企业与开源</h2>
+      <p class="p-heading--four">Ubuntu的发布者。<br /> 安全，支持。代管服务。</p>
+      <p>
+        <a class="p-button--positive js-invoke-modal" href="/contact">
```

Comment:
```suggestion
        <a class="p-button--positive" href="/contact">
```

Reply:
Removed all the js-invoke-modal classes because it hasn't been used.